app/presenters/organisations/what_we_do_presenter.rb

```diff
@@ -14,10 +14,11 @@ def share_links
       links = []
 
       org.social_media_links.each do |link|
+        link_has_cta = ["Sign up", "Follow", "Watch", "Read"].any? { |cta| link["title"].include?(cta) }
```

Comment:
@danacotoran are these Icon Content's likely to change? Probably not, but I'm wondering if they update and this gets skipped.  Eg the gem has ["Get email alerts"](https://components.publishing.service.gov.uk/component-guide/subscription-links) rather than "Sign up"

Reply:
Yep as you've pointed out, it's possible the links might be updated to use different wording.  But I've checked a fairly large number of organisation pages with social media links and from what I could see if any "call to action" verbs are used for their social media links, it's usually the ones I listed.

As you can tell, this is somewhat of a hack/workaround to make sure that the issue is fixed in most cases.
Since we're dealing with language, which is so fluid, I think it would be tricky to implement a catch-all solution, so my objective with this was more in the vein of "fix most instances where the issue would occur" than "make it perfect".

Reply:
>  I think it would be tricky to implement a catch-all solution, so my objective with this was more in the vein of "fix most instances where the issue would occur" than "make it perfect".

Fair.